### DISCUSSION TITLE
Publish fix for regression in 4.7.2

### DISCUSSION THREAD

**Discussion by tmoschou** [spotbugs/spotbugs#2215] [General]
Hello.

Is it possible to publish a new release that includes the [fix](https://github.com/spotbugs/spotbugs/issues/2174) for the [regression](https://github.com/spotbugs/spotbugs/pull/2141#issuecomment-1243643651) in 4.7.2 or advise on the expected timeframe.

The regression unfortunately spams our build logs with thousands of lines of stack traces. We are currently downgraded to 4.7.1, but we would like to also include the fix for https://github.com/spotbugs/spotbugs/issues/2145 in 4.7.2 also, so that we can remove the current exclusion for Groovy classes. 

Thanks

**Comment by tmoschou** [comment]
I just saw v4.7.3 was released, much appreciated 🙏

**Comment by hazendaz** [comment]
closing as resolved.
